examples/rails-webpacker/app/javascript/src/tailwind.css

```diff
@@ -0,0 +1,3 @@
+@import "tailwindcss/base";
```

Comment:
If you use `PurgeCSS` it will remove all `normalize.css` styling. [This post give a solution](https://frontstuff.io/how-i-dropped-250-kb-of-dead-css-weight-with-purgecss) but I'm not sure how to apply it in the case of Webpacker

Reply:
Oh, this just solved it
![image](https://user-images.githubusercontent.com/49499/72681390-5ce88f00-3ac3-11ea-8e31-1247c989cd28.png)